Parsers/ASimRegistryEvent/Parsers/ASimRegistry.yaml

```diff
@@ -26,6 +26,7 @@ Parsers:
   - _ASim_RegistryEvent_MicrosoftSysmon
   - _ASim_RegistryEvent_MicrosoftWindowsEvent
   - _ASim_RegistryEvent_SentinelOne
+  - _ASim_RegistryEvent_TrendMicroVisionOne
 ParserQuery: |
   let DisabledParsers=materialize(_GetWatchlist('ASimDisabledParsers') | where SearchKey in ('Any', 'ExcludeASimRegistry') | extend SourceSpecificParser=column_ifexists('SourceSpecificParser','') | distinct SourceSpecificParser| where isnotempty(SourceSpecificParser));
   let ASimBuiltInDisabled=toscalar('ExcludeASimRegistryEventBuiltIn' in (DisabledParsers) or 'Any' in (DisabledParsers));
```

Comment:
@jayeshprajapaticrest @niralishah-crest Please include TrendMicro Caller under ParserQuery property

Reply:
also, there are merge conflicts

Reply:
@vakohl We have added TrendMicro under ParserQuery property